{
  "uid" : "589e57d185f9ea8f",
  "name" : "test_invalid_email",
  "fullName" : "tests.test_forgot_password.TestForgotPassword#test_invalid_email",
  "historyId" : "415bbb87c1005e344ea1c6b819fb2117",
  "time" : {
    "start" : 1604104753025,
    "stop" : 1604104755740,
    "duration" : 2715
  },
  "description" : "Forgot Password feature test with invalid email address",
  "descriptionHtml" : "<p>Forgot Password feature test with invalid email address</p>\n",
  "status" : "broken",
  "statusMessage" : "TypeError: selenium.webdriver.remote.webdriver.WebDriver.find_element() argument after * must be an iterable, not WebElement",
  "statusTrace" : "self = <tests.test_forgot_password.TestForgotPassword object at 0x7f57932557c0>\ncreate_driver = <selenium.webdriver.remote.webdriver.WebDriver (session=\"12a332e2e66683da4e75dc9fd4a863ed\")>\n\n    @allure.description(\"Forgot Password feature test with invalid email address\")\n    def test_invalid_email(self, create_driver):\n        ap = AboutPage()\n        lp = LoginPage()\n        fp = ForgotPasswordPage()\n        ap.click_login_link()\n        lp.click_forgot_password()\n        fp.send_password_reset_link(\"something@gmail.com\")\n>       assert_that(fp.get_invalid_email_msg()).is_equal_to(self._error_msg)\n\ntests/test_forgot_password.py:38: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/forgot_password_page.py:23: in get_invalid_email_msg\n    return self.get_text(self._driver.find_element(*self.ERROR_MSG))\npages/base_page.py:40: in get_text\n    el = self._wait.until(expected_conditions.visibility_of_element_located(webelement))\n/opt/hostedtoolcache/Python/3.9.0/x64/lib/python3.9/site-packages/selenium/webdriver/support/wait.py:71: in until\n    value = method(self._driver)\n/opt/hostedtoolcache/Python/3.9.0/x64/lib/python3.9/site-packages/selenium/webdriver/support/expected_conditions.py:128: in __call__\n    return _element_if_visible(_find_element(driver, self.locator))\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\ndriver = <selenium.webdriver.remote.webdriver.WebDriver (session=\"12a332e2e66683da4e75dc9fd4a863ed\")>\nby = <selenium.webdriver.remote.webelement.WebElement (session=\"12a332e2e66683da4e75dc9fd4a863ed\", element=\"0d84df70-f9d7-4437-bc4d-f0aae58017a4\")>\n\n    def _find_element(driver, by):\n        \"\"\"Looks up an element. Logs and re-raises ``WebDriverException``\n        if thrown.\"\"\"\n        try:\n>           return driver.find_element(*by)\nE           TypeError: selenium.webdriver.remote.webdriver.WebDriver.find_element() argument after * must be an iterable, not WebElement\n\n/opt/hostedtoolcache/Python/3.9.0/x64/lib/python3.9/site-packages/selenium/webdriver/support/expected_conditions.py:411: TypeError",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "prep_properties",
    "time" : {
      "start" : 1604104650312,
      "stop" : 1604104650312,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "create_driver",
    "time" : {
      "start" : 1604104749341,
      "stop" : 1604104752449,
      "duration" : 3108
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "_session_faker",
    "time" : {
      "start" : 1604104650223,
      "stop" : 1604104650312,
      "duration" : 89
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "testStage" : {
    "description" : "Forgot Password feature test with invalid email address",
    "status" : "broken",
    "statusMessage" : "TypeError: selenium.webdriver.remote.webdriver.WebDriver.find_element() argument after * must be an iterable, not WebElement",
    "statusTrace" : "self = <tests.test_forgot_password.TestForgotPassword object at 0x7f57932557c0>\ncreate_driver = <selenium.webdriver.remote.webdriver.WebDriver (session=\"12a332e2e66683da4e75dc9fd4a863ed\")>\n\n    @allure.description(\"Forgot Password feature test with invalid email address\")\n    def test_invalid_email(self, create_driver):\n        ap = AboutPage()\n        lp = LoginPage()\n        fp = ForgotPasswordPage()\n        ap.click_login_link()\n        lp.click_forgot_password()\n        fp.send_password_reset_link(\"something@gmail.com\")\n>       assert_that(fp.get_invalid_email_msg()).is_equal_to(self._error_msg)\n\ntests/test_forgot_password.py:38: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/forgot_password_page.py:23: in get_invalid_email_msg\n    return self.get_text(self._driver.find_element(*self.ERROR_MSG))\npages/base_page.py:40: in get_text\n    el = self._wait.until(expected_conditions.visibility_of_element_located(webelement))\n/opt/hostedtoolcache/Python/3.9.0/x64/lib/python3.9/site-packages/selenium/webdriver/support/wait.py:71: in until\n    value = method(self._driver)\n/opt/hostedtoolcache/Python/3.9.0/x64/lib/python3.9/site-packages/selenium/webdriver/support/expected_conditions.py:128: in __call__\n    return _element_if_visible(_find_element(driver, self.locator))\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\ndriver = <selenium.webdriver.remote.webdriver.WebDriver (session=\"12a332e2e66683da4e75dc9fd4a863ed\")>\nby = <selenium.webdriver.remote.webelement.WebElement (session=\"12a332e2e66683da4e75dc9fd4a863ed\", element=\"0d84df70-f9d7-4437-bc4d-f0aae58017a4\")>\n\n    def _find_element(driver, by):\n        \"\"\"Looks up an element. Logs and re-raises ``WebDriverException``\n        if thrown.\"\"\"\n        try:\n>           return driver.find_element(*by)\nE           TypeError: selenium.webdriver.remote.webdriver.WebDriver.find_element() argument after * must be an iterable, not WebElement\n\n/opt/hostedtoolcache/Python/3.9.0/x64/lib/python3.9/site-packages/selenium/webdriver/support/expected_conditions.py:411: TypeError",
    "steps" : [ {
      "name" : "Click Login link",
      "time" : {
        "start" : 1604104753025,
        "stop" : 1604104754261,
        "duration" : 1236
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    }, {
      "name" : "Click Forgot Password link",
      "time" : {
        "start" : 1604104754262,
        "stop" : 1604104754762,
        "duration" : 500
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    }, {
      "name" : "Send password reset link to email address: 'something@gmail.com'",
      "time" : {
        "start" : 1604104754763,
        "stop" : 1604104755687,
        "duration" : 924
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ {
        "name" : "email",
        "value" : "'something@gmail.com'"
      } ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : true
    }, {
      "name" : "Get invalid email message",
      "time" : {
        "start" : 1604104755687,
        "stop" : 1604104755740,
        "duration" : 53
      },
      "status" : "broken",
      "statusMessage" : "TypeError: selenium.webdriver.remote.webdriver.WebDriver.find_element() argument after * must be an iterable, not WebElement\n",
      "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.9.0/x64/lib/python3.9/site-packages/allure_commons/_allure.py\", line 157, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/selenium-python/selenium-python/pages/forgot_password_page.py\", line 23, in get_invalid_email_msg\n    return self.get_text(self._driver.find_element(*self.ERROR_MSG))\n  File \"/home/runner/work/selenium-python/selenium-python/pages/base_page.py\", line 40, in get_text\n    el = self._wait.until(expected_conditions.visibility_of_element_located(webelement))\n  File \"/opt/hostedtoolcache/Python/3.9.0/x64/lib/python3.9/site-packages/selenium/webdriver/support/wait.py\", line 71, in until\n    value = method(self._driver)\n  File \"/opt/hostedtoolcache/Python/3.9.0/x64/lib/python3.9/site-packages/selenium/webdriver/support/expected_conditions.py\", line 128, in __call__\n    return _element_if_visible(_find_element(driver, self.locator))\n  File \"/opt/hostedtoolcache/Python/3.9.0/x64/lib/python3.9/site-packages/selenium/webdriver/support/expected_conditions.py\", line 411, in _find_element\n    return driver.find_element(*by)\n",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : true,
      "hasContent" : true
    } ],
    "attachments" : [ {
      "uid" : "4db6ba53dc3bac8d",
      "name" : "screenshot on failure: 31/10/2020, 00:39:12",
      "source" : "4db6ba53dc3bac8d.png",
      "type" : "image/png",
      "size" : 519743
    }, {
      "uid" : "70fa08c25a5eef3d",
      "name" : "screenshot on failure: 31/10/2020, 00:39:15",
      "source" : "70fa08c25a5eef3d.png",
      "type" : "image/png",
      "size" : 42625
    } ],
    "parameters" : [ ],
    "stepsCount" : 4,
    "attachmentsCount" : 2,
    "shouldDisplayMessage" : true,
    "hasContent" : true
  },
  "afterStages" : [ {
    "name" : "create_driver::0",
    "time" : {
      "start" : 1604104756163,
      "stop" : 1604104756217,
      "duration" : 54
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "labels" : [ {
    "name" : "severity",
    "value" : "critical"
  }, {
    "name" : "story",
    "value" : "Forgot Password Feature's Functionality"
  }, {
    "name" : "epic",
    "value" : "Security"
  }, {
    "name" : "tag",
    "value" : "security"
  }, {
    "name" : "parentSuite",
    "value" : "tests"
  }, {
    "name" : "suite",
    "value" : "test_forgot_password"
  }, {
    "name" : "subSuite",
    "value" : "TestForgotPassword"
  }, {
    "name" : "host",
    "value" : "fv-az68-995"
  }, {
    "name" : "thread",
    "value" : "4814-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "tests.test_forgot_password"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "critical",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "tags" : [ "security" ]
  },
  "source" : "589e57d185f9ea8f.json",
  "parameterValues" : [ ]
}